sa/sa.go

```diff
@@ -438,6 +438,18 @@ func (ssa *SQLStorageAuthority) UpdateRegistration(ctx context.Context, reg core
 	return nil
 }
 
+// recordIssuedNames updates the issuedNames table to track issuance for the
```

Comment:
I think this function got left behind; it now has an empty body.

Reply:
Oops! Thanks for catching this. Forgot to clean it up once lifting things into a separate function stopped making sense. Fixed in 1d35ca3.